common/cards/card-plugins/hermits/_hermit-card.js

```diff
@@ -59,7 +59,7 @@ class HermitCard extends Card {
 	 * @param {AttackModel} attack
 	 */
 	onAttack(game, instance, attack) {
-		// default is do nothing
+		attack.addDamage(attack.type === 'primary' ? this.primary.damage : this.secondary.damage)
```

Comment:
remember there can also be zero attacks, meaning that we might add no damage.

Reply:
The cards that use Zero attack are going to override onAttack, this is just for common hermits.